jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -179,6 +179,8 @@
 	/*[PR CMVC 125822] Move RAM class fields onto the heap to fix hotswap crash */
 	private transient ProtectionDomain protectionDomain;
 	private transient String classNameString;
+	/* https://github.ibm.com/runtimes/openj9/issues/496 Calling Class.toString() causes 1% throughput cost */
```

Comment:
I don't think it's appropriate to include a private URL in an open-source project.